sql/core/src/main/scala/org/apache/spark/sql/execution/python/streaming/TransformWithStateInPySparkPythonRunner.scala

```diff
@@ -245,7 +245,6 @@ abstract class TransformWithStateInPySparkPythonBaseRunner[I](
 class TransformWithStateInPySparkPythonPreInitRunner(
     func: PythonFunction,
     workerModule: String,
-    timeZoneId: String,
```

Comment:
It is only used for constructing `TransformWithStateInPySparkStateServer`, so `timeZoneId` is no longer needed either.